### DISCUSSION TITLE
¡Hola desde Colombia! 🇨🇴

### DISCUSSION THREAD

**Discussion by zejiran** [gittogethers/octogatosconf22#130] [Where in the world?]
Emocionado por conocer y conectar con las experiencias de esta comunidad. 😄

**Comment by AndreaGriffiths11** [comment]
Hola! paisano! saludos

**Reply by flutterinapps** [reply]
Hola

**Comment by BrayanGuerreroXD** [comment]
Hola como están parceros?

**Comment by juanmrad** [comment]
Hola paisano! 😄

**Comment by actually-not-an-username** [comment]
Hola paisanos!!

**Comment by AndreaGriffiths11** [comment]
¡Este es el combo! ¡Muchas gracias por estar aquí, presentes!

**Comment by mrincodi** [comment]
¡Hola a todos! Saludos de un samario en Austin, Texas, EE. UU.

**Comment by Moogh** [comment]
Buenas noches a todos, ¿listos para la conferencia?
